libs/journeys/ui/src/components/StepFooter/NameAndLocation/NameAndLocation.tsx

```diff
@@ -0,0 +1,49 @@
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+
+import { ReactElement } from 'react'
+import { useJourney } from '../../../libs/JourneyProvider'
+import { getJourneyRTL } from '../../../libs/rtl'
+
+interface NameAndLocationProps {
+  name: string
+  location?: string
+}
+
+export const NameAndLocation = ({
+  name,
+  location = ''
+}: NameAndLocationProps): ReactElement => {
+  const { journey } = useJourney()
+  const { rtl } = getJourneyRTL(journey)
+
+  return (
+    <Stack
+      className="name-and-location"
+      sx={{
+        flexDirection: 'row',
+        alignItems: 'center',
+        flexWrap: 'none'
+      }}
+    >
+      <Typography
+        variant="body2"
+        sx={{
+          whiteSpace: 'nowrap',
+          overflow: 'clip',
+          textOverflow: 'ellipsis',
+          color: { xs: 'primary.main', lg: 'white' },
```

Comment:
```suggestion
          color: { xs: 'primary.main', lg: 'background.default' },
```

if it's just meant to be white, can use token?

Reply:
ignore this one actually

Reply:
keeping this consistent with how the step footer title is made since that PR and colors has been approved by UI